islands/src20/SRC20Section.tsx

```diff
@@ -112,10 +111,6 @@ export function SRC20Section(
               )
           ))}
         </div>
-
```

Comment:
This may remove the view all from other pages that use this component. Should be a dynamic display??

Reply:
yes, of course
we can do that.
then could you please let me know the page that I need to show the view all ?